.travis.yml

```diff
@@ -139,7 +139,7 @@ script:
       echo 'image.cmap : viridis' >> $MPL_RC_DIR/matplotlibrc;
       cd $INSTALL_DIR/docs/iris;
       echo `make clean html`;
-      make doctest;
```

Comment:
@pp-mo surely you need this semicolon ... or is it complaining otherwise?

Reply:
I think you don't need the semicolon on the last line of an "if" block, if there is a separating newline.
But thats all moot since the real problem is as @QuLogic identified, so I will put this back the way it was, 
for minimal change if nothing else ...